# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/wifi/win_wifiScanner.h
# Commit: 35c408b9531d
# Full Hash: 35c408b9531db01c71dee288cab0a02a633fb535
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-13 21:42:25
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Refactor wifi scanning to consolidate generic code r=necko-reviewers,kershaw,valentin
#   
#   Reorganizes the wifi-scanning code and makes more of it platform-generic
#   to ease the transition from polling the wifi to usually scanning only on
#   network changes.  This is essentially just moving files/code around and
# ==============================================================================

diff -r 029be11fa3d8 -r 35c408b9531d netwerk/wifi/win_wifiScanner.h
--- a/netwerk/wifi/win_wifiScanner.h	Sat May 13 00:11:51 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,32 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#pragma once
-
-// Moz headers (alphabetical)
-#include "mozilla/UniquePtr.h"
-#include "nsCOMArray.h"
-#include "win_wlanLibrary.h"
-
-class nsWifiAccessPoint;
-
-class WinWifiScanner final {
- public:
-  WinWifiScanner();
-  ~WinWifiScanner();
-
-  /**
-   * GetAccessPointsFromWLAN
-   *
-   * Scans the available wireless interfaces for nearby access points and
-   * populates the supplied collection with them
-   *
-   * @param accessPoints The collection to populate with available APs
-   * @return NS_OK on success, failure codes on failure
-   */
-  nsresult GetAccessPointsFromWLAN(nsCOMArray<nsWifiAccessPoint>& accessPoints);
-
- private:
-  mozilla::UniquePtr<WinWLANLibrary> mWlanLibrary;
-};